CRBM-Viz/angular.json

```diff
@@ -7,7 +7,8 @@
       "projectType": "application",
       "schematics": {
         "@schematics/angular:component": {
-          "style": "sass"
+          "style": "sass",
+          "styleext": "scss"
```

Comment:
This will break the styles for all the components that currently use sass. Perhaps the theme can be changed to sass instead of scss?

Reply:
Nevermind. When merged, this seems to work

Reply:
The components can still use sass. I changed to use a custom theme rather than the default theme. The custom theme is in scss.